src/Microsoft.AspNetCore.Server.Kestrel/Microsoft.AspNetCore.Server.Kestrel.csproj

```diff
@@ -12,6 +12,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="$(AspNetCoreVersion)" />
```

Comment:
This is to not break upstream components that might have been relying on Kestrel depending on Hosting. I don't want to change that as part of this. The goal here was to clean up some of the dependencies after the initial change.